docs/sphinx/user_guide/config_options.rst

```diff
@@ -101,7 +101,7 @@ and their default settings:
       =========================   ======================
       Variable                    Default
       =========================   ======================
-      ENABLE_WARNINGS_AS_ERRORS   Off
+      RAJA_ENABLE_WARNINGS_AS_ERRORS   Off
```

Comment:
I may be wrong, but think these changes are going to break the sphinx docs formatting. As I understand it, the '=' row delimeters need to extend at least as far as the test in each row. Adding more '=' so the columns align should fix it. Have you looked at the regenerated docs with these changes?

Reply:
Indeed, running text into the column margins breaks the sphinx build. I pushed changes to fix.

Reply:
Thanks Rich!